backend/codingchallenge/api/views/authentication.py

```diff
@@ -40,6 +41,11 @@ def post(self, request, *args, **kwargs):
             user = authenticate(request, username=username, password=password)
 
             if user:
+                if user.application.expiry > time.time():
```

Comment:
Doesnt "expiry > current time" mean, that the expiry happens AFTER the current time?
Since time.time() returns the current time since a specific date, a greater time value means, that it will happen further in the future.
If my thoughts are correct, please change the condition from > to <

Reply:
Thank you for your hint. You are right. I will change it.